playground/frontend/playground_components/lib/src/controllers/playground_controller.dart

```diff
@@ -88,6 +89,8 @@ class PlaygroundController with ChangeNotifier {
     final result = SnippetEditingController(sdk: sdk);
     _snippetEditingControllers[sdk] = result;
     result.addListener(notifyListeners);
+    GetIt.instance.get<FeedbackController>().eventSnippetContext =
+        result.eventSnippetContext;
```

Comment:
A call to `_getOrCreateSnippetEditingController` does not necessarily mean the new editor is active. So the feedback's `eventSnippetContext` should not be set here.